#153009245 configure eslint

oti4me opened the pull request:
#### What does this PR do?

Configure ESLint

#### Description of Task to be completed?

- Install ESLint and other dependencies
- Run eslint --init and choose the airbnb popular style guide
- Configure the generated .eslintrc.yml file to include the airbnb style rules

#### How should this be manually tested?

- Clone online repository
- Navigate to the cloned directory
- Add ESLint as extension to your favourite text editor
- Run npm install
- Open the codebase in your favourite text editor to see linting errors
- Run: npm run eslint on your command prompt to test for linting errors

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

- story type: chore
- story id: 153009245
- story title: Configure ESLint

**Screenshots (if appropriate)**

<img width="1440" alt="screen shot 2017-11-21 at 7 16 26 pm" src="https://user-images.githubusercontent.com/14099353/33089363-a195331a-cef0-11e7-9adb-59b75f370a24.png">

Commit:
chore(stabilizeApplication): install all dependencies
- create .env file
- create .env.example file on how to use .env file
[Delivers #1530090342]

temilaj commented on npm-debug.log:
```diff
@@ -0,0 +1,48 @@
+0 info it worked if it ends with ok
```
Why do you we have this file in our repo?

temilaj reviewed (changes requested):
I'm curious to know why we're using the .yml configuration file. How well will this work with houndci?

Commit:
chore(eslint): configure eslint

- install eslint
- install eslint-config-airbnb
- install eslint-plugin-import
- import eslint-plugin-react
- install eslint-plugin-jsx-a11y
-run eslint --init to initialize eslint
- select popular style guide, then select airbnb
- configure the generated  eslintrc file to use airbnb style rules

[finished #153009245]

oti4me commented:
@temilaj I have made the changes you pointed. I have convert the .eslintrc.yml file to .eslintrc file and have remove the npm-debug.log file from the repository.

temilaj reviewed (changes requested):
Good work improving your PR. 
However we now have 5 commits that are essentially the same. Take time out to squash these commits into 1, and then raise another PR
